src/locales/es.ts

```diff
@@ -238,11 +237,11 @@ export default {
     saintJuanDiego: 'San Juan Diego',
     saintJuanDiegoCuauhtlatoatzin: 'San Juan Diego Cuauhtlatoatzin',
     saintLeoTheGreatPopeAndDoctor: 'San Leo el Magno, Papa y Doctor',
-    saintLorenzoRuizAndCompanionsMartyrs: 'Saint Lorenzo Ruiz y Companions, Mártires',
+    saintLorenzoRuizAndCompanionsMartyrs: 'San Lorenzo Ruiz y Compañeros, Mártires',
     saintLouisSaintJosephOfCalasanzPriest: 'San Luis/San José de Calasanz, Sacerdote',
     saintMargaretMaryAlacoque: 'Santa Margarita María Alacoque',
```

Comment:
Should it be `Santa Margarita María Alacoque` or `Santa Margarita María de Alacoque`? There are two (duplicate) keys of her celebration and you have localised it two different ways.

Reply:
Both are accepted although I think the most accurate would be "de Alacoque", will change that.